src/App.test.js

```diff
@@ -1,9 +0,0 @@
-import React from 'react';
```

Comment:
Should we keep this test and imports? I dont see why not.

Reply:
It's still there. I think it's showing red because I moved src/App.test.js into src/App/App.test.js. Same with App.js.

Reply:
Sounds good merging in now